src/test/java/com/echoserver/test/EchoServerTest.java

```diff
@@ -1,13 +1,35 @@
-
 package com.echoserver.test;
 
 import com.echoserver.EchoServer;
 import org.junit.Test;
 
+import java.io.*;
+
 public class EchoServerTest {
+    private EchoServer server = null;
```

Comment:
For info, Java will automatically default objects to null when they are class variables, so you don't necessarily have to assign them to null here.